docs/references/debugging-sdk-tools.md

```diff
@@ -0,0 +1,83 @@
+# Debugging SDK Tools
+Now that you've built your first SDK Tool, you likely want to confirm that it's properly installed and that the code you've written is being executed. This guide reviews the places you should check for correctness and how to verify that your plugin is being correctly built and executed. We recommended to follow these steps when your tool doesn't behave as expected. Additionally, always rebuild using the `create-yxi` command when iterating and ensure that changes are being made to your plugin code.  
+
+
+## Table of Contents
+1. After Running in Designer
+2. Running Plugin via Command Line
+3. Simple Ways to Print and Check Values
+4. After `create-yxi`
+5. After Installing the YXI
+
+
+## 1. After Running in Designer
+If, after you run the plugin in Designer, the tool does not yield the expected results, you can check the logs via this method:
+
+> Check for PythonSDK.log and enable Debug logging:
+
+> - Tool logs are written to a file called PythonSDK.log located in Local AppData. For example, `%LOCALAPPDATA%\Alteryx\Log\PythonSDK.log`.
```

Comment:
I'd make "Local AppData" one word, all caps, and quote it in backticks, showing that it's a variable.

eg `LOCALAPPDATA`

Reply:
Also worth noting, below, in section 5, we use `%APPDATA`. To be fair, this is the one I've always used.  There is likely some overlap, but that could be confusing to the reader.